### ISSUE TITLE
Need better error if trying to set property that already exists

### ISSUE THREAD

**Issue by dustinfarris** [ember-redux/ember-redux#55]
e.g., this would error:

```hbs
{{my-component name="Something"}}
```

```js
const stateToComputed = () => {
  return {
    name: "Something else"
  }
};
```

Right now the error is `can't set read-only property "name" on object`.  We could probably trap this provide some guidance on what likely went wrong.

**Comment by toranb** [comment]
@dustinfarris so the issue is the attr conflict - can you write up something to look for a conflict in the init/or didRecieveAttrs (not sure which is the best place w/out looking at a stacktrace myself)

I see this as a solid quality of life improvement @dustinfarris and others seem to agree :)

**Comment by dustinfarris** [comment]
Yes, I'll take a swing at this

**Comment by toranb** [comment]
@dustinfarris you the man! +100 !!!

**Comment by toranb** [comment]
@brettburley fixed this in #117
